### PR TITLE
fix(registry): explain PORT is required when using private registry information

### PR DESCRIPTION
**Pull request by helgi**

### CODE REVIEW

File: src/applications/using-docker-images.md [hunk 1]
```diff
@@ -78,6 +78,11 @@ tool like [jq][] and then used in the password field in `deis registry:set`. For
 deis registry:set username=_json_key password="$(cat google_cloud_cred.json | jq -c .)"
 ```
 
+When using private registry the docker images are no longer pulled into the Deis Internal Registry via
```
Comment (arschles) [inline src/applications/using-docker-images.md]: This should read: "When using __a__ public registry..."

File: src/applications/using-docker-images.md [hunk 2]
```diff
@@ -78,6 +78,11 @@ tool like [jq][] and then used in the password field in `deis registry:set`. For
...
+the Deis Workflow Controller but rather is managed by Kubernetes. This will increase security and overall speed
+however the `port` information can no longer be discovered. Instead the `port` information can be set via
```
Comment (arschles) [inline src/applications/using-docker-images.md]: put a comma between `speed` and `however`

**Comment by arschles** [comment]
docs LGTM assuming https://github.com/deis/workflow/pull/289#discussion_r66313671 and https://github.com/deis/workflow/pull/289#discussion_r66313726 are done

**Comment by helgi** [comment]
Fixed

**Pull request merged**